lib/client.js

```diff
@@ -216,8 +217,20 @@ exports = module.exports = class Tgfancy extends TelegramBot {
 
         // We are now performing 'on ready' actions.
         // Let's set the correct webhook when on Openshift
-        if (this.options._openshift) {
+        if (this.options._openshift && process.env.OPENSHIFT_APP_UUID) {
```

Comment:
Yes, you're right, I confused `this.options._openshift`with `opts.openshiftWebHook`.

What to do with the commit? Rebase it out now, or revert `--fixup` it, to be rebased out before merging?